### ISSUE TITLE
Windows support <3

### ISSUE THREAD

**Issue by mpfaffenberger** [ddworken/hishtory#12]
Hi! I'm super excited about this and I've already got it running on all of my *nix. 

When trying to run with `git bash` I get:

`No hishtory binary for system=Windows, machine=AMD64!`

I assume I could probably build from scratch right? But it would be awesome if binary was available.

**Comment by ddworken** [comment]
Honestly, I've never used `git bash` so I have no idea if it would work on Windows. Though I'm guessing it wouldn't since Windows likely has different paths for the various config files. I'm not opposed to this, but since I personally don't use Windows I'm not likely to prioritize this in the short term. 

And just to put the alternative out there, I believe it should work fine in [WSL](https://learn.microsoft.com/en-us/windows/wsl/about).

**Comment by mpfaffenberger** [comment]
Yeah no worries, figured I'd throw it out there. I don't blame anyone for not using windoze.

**Comment by Ama1999** [comment]
Hmmm... I might take a peek at it soon™️

**Comment by ddworken** [comment]
Awesome! Contributions are very welcome. :)
